hathorlib/client.py

```diff
@@ -102,6 +102,7 @@ async def get_block_template(self, address: Optional[str] = None) -> BlockTempla
 
             # Build block.
             blk = Block()
+            blk.signal_bits = data['signal_bits']
```

Comment:
Should we add a test that would fail without this change?

Reply:
Done in https://github.com/HathorNetwork/python-hathorlib/pull/32/commits/8988601cd1291bebe3bd2bf4a79ad0c3886f8a16

Reply:
Thanks!